src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -713,7 +713,9 @@ class TableIPRDescription(CommonIPR):
 @attr.s(frozen=True, slots=True)
 class IPRModelsDescription:
     """
-    :cvar Dict[str, Union[str, IPRDescription]] tables:
+    :type tables Dict[str, Union[str, IPRDescription]]
```

Comment:
I don't think we need this `:type` directive at all.

Where is `tables` declared? I only see `linear_models` and `table_models` below.

Reply:
Ops, typo... 

We may need to choose a pattern for our docstring. By default `sphinx` uses `:type` for typing.
With the google pattern we could use the type annotations. 
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
```
def function_with_pep484_type_annotations(param1: int, param2: str) -> bool:
    """Example function with PEP 484 type annotations.

    Args:
        param1: The first parameter.
        param2: The second parameter.

    Returns:
        The return value. True for success, False otherwise.

    """

Reply:
Sphinx standard also reads type annotations... that's what we use in pytest. 😁 

In summary, both Google and Sphinx notations read type annotations, so we can drop them from the docs. 👍

Reply:
Ohh cool! I take a look in pytest `conf.py` and there is used `autodoc_typehints = "description"`. I used it here and don't have the same effect. I'll review all types and try to remove then from docstring.

But looking at pytest doc, some places it don't have annotations in the docs:
https://docs.pytest.org/en/6.2.x/reference.html?highlight=mark#node

Reply:
>  I used it here and don't have the same effect. I'll review all types and try to remove then from docstring.

Hmm what do you mean? The first phrase conveys that it didn't work, but the second phrase to me indicates it did work and you will review the other types... can you clarify please?

> But looking at pytest doc, some places it don't have annotations in the docs

Probably they are not type-annotated yet, but don't worry about it.

Reply:
Yeah, it didn't work, but I will review all annotations and remove then from docs to see how it will behave.

Reply:
It's more a matter of concept than rendering... those attributes are instance vars, not parameters. If the rendering is not great/has warnings that's another problem to solve.